internal/controllers/analyzer/analyzer.go

```diff
@@ -172,20 +172,31 @@ func (a *Analyzer) runAnalysis() (totalVulns int, err error) {
 		return 0, err
 	}
 	a.startDetectVulnerabilities(langs)
-	return a.sendAnalysisAndStartPrintResults()
+	if err = a.sendAnalysis(); err != nil {
+		loggerSendError(err)
```

Comment:
Why not just use `logger.LogStringAsError(fmt.Sprintf("\n[HORUSEC] %v", err))`

Since we are just logging here we don't need a new method for just logging. Also, it is unnecessary to double-check that the error is not nil